### PR TITLE
Change submission status

### LINKED ISSUE #5: Change submission status
Add changing of a submission status

### PR DESCRIPTION
**Pull request by thomasyu888**
- [x] fixes #5
- [x] Did you run your tests locally (instructions [here](https://github.com/Sage-Bionetworks/challengerutils/blob/master/CONTRIBUTING.md)?
- [x] Did you add a good description about your changes or additions?

This is the R convenience function for changing a submission status.

### CODE REVIEW

File: R/submission.R [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#' @param status Change a submision to this status. Default to RECEIVED.
+.change_submission_status <- function(submissionid, status = "RECEIVED") {
+  change_status <- challengeutils$utils$change_submission_status
+  change_status(syn = syn, submissionid = submissionid, status = status)
+}
+
+#' Changes a status of a submission
+#'
+#' @param submissionid Id of a submission
+#' @param status Change a submision to this status. Default to RECEIVED.
```
Comment (allaway) [inline R/submission.R:13]: obviously i guess you can set this to anything but should this link to docs that indicate what the status options are and what they mean?

**Review by allaway — CHANGES_REQUESTED** [review]
Looks good to me! Made one comment about the documentation.

File: R/submission.R [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+#' Changes a status of a submission
+#'
+#' @param submissionid Id of a submission
+#' @param status Change a submission to this status. Defaults to RECEIVED.
+#' Here are a list of available statuses.
+#' https://rest-docs.synapse.org/rest/org/sagebionetworks/evaluation/model/SubmissionStatusEnum.html
+#'
+#' @return Updated submisison status
```
Comment (thomasyu888) [inline R/submission.R:19]:
```suggestion
#' @return Updated submission status
```

File: README.md [hunk 3]
```diff
@@ -30,14 +30,21 @@ The functions will mimic the Python implementation of `challengeutils`.  Please
...
+challengerutils::syn_login()
 ```
 
 * Query Evaluation Queue
 ```
-challengerutils::syn_login()
 leaderboard_results <- challengerutils::evaluation_queue_query('select * from evaluation_9614194')
 ```
 
+* Change submisison status
```
Comment (thomasyu888) [inline README.md:42]:
```suggestion
* Change submission status
```

**Review by allaway — APPROVED** [review]

**Pull request merged**